doc/data/messages/i/invalid-unary-operand-type/good.py

```diff
@@ -1 +1,3 @@
-# This is a placeholder for correct code for this message.
+fruit = 1
+worm = 10
+worm -= fruit
```

Comment:
```suggestion
cherries = 10
eaten_cherries = 2
cherries -= eaten_cherries
```

Reply:
Yes, both examples make much more sense. :) Thank you very much.